Frontend/app/tests/example.ts

```diff
@@ -0,0 +1,31 @@
+// A sample QUnit test
+
+/*QUnit.test("equal test", function (assert) {
+  assert.equal( 0, 0, "Zero, Zero; equal succeeds" );
+  assert.equal( "", 0, "Empty, Zero; equal succeeds" );
+  assert.equal( "", "", "Empty, Empty; equal succeeds" );
+  assert.equal( 0, false, "Zero, false; equal succeeds" );
+});*/
+
+var participants = require("../pages/enterprises/participants.js");						//Access Participant variables and functions
+var newParticipant = new participants.Participant("Jim");
+QUnit.test("Participant Test", function (assert) {
+    assert.equal( newParticipant.strName, "Jim", "Participant's name is indeed Jim" ); 	//Assert that the participant's name is Jim.
+});
+
+
+var participant1 = new participants.Participant("Ben10");
+var participant2 = new participants.Participant("Kevin11");
+var participant3 = new participants.Participant("Bob12");
+var participantList = [participant1, participant2, participant3];
+var enterprises = require("../pages/enterprises/enterprise.js");
```

Comment:
Same as above - as far as I can tell the enterprise class isn't defined anywhere, so how would this work? Ideally at all stages the Master branch should compile and all tests should pass.